lib/beeline/health_checker.ex

```diff
@@ -36,6 +36,7 @@ defmodule Beeline.HealthChecker do
             the producer
           * `:head_position` (integer) - the stream position of the head
             (the latest event) of the EventStoreDB stream
+          * `:auto_subscribe` (boolean) - whether the producer auto subscribes on startup
```

Comment:
```suggestion
          * `:auto_subscribe` (boolean) - the value of the producer's `:auto_subscribe?`
            flag at the time of polling
```

just to clarify whether this is a static value or polled dynamically (it's the latter)